dbt_marketplace_base/views/dbt_instance_view.xml

```diff
@@ -16,19 +16,9 @@
                                 <field name="name" />
                             </h1>
                         </div>
-                       <group string="Option" attrs="{'invisible':[('connected','=',False)]}">
-                        </group>
                         <group string="Connection">
-                            <!-- <field name="dbt_test"/> -->
                             <field name="active"/>
```

Comment:
other fields of base_template needs to be added.
Also can we think of some way to give link to connected Instance (Not only bol but for any other projects too)

Reply:
Do comments your thoughts on how we can show linked instance before actually working on it @devang-dreambits

Reply:
Opening an action form a button can help us. But we need 1 more field for saving action of the connector class....ex 

'child_action_id': self.env.ref('dbt_bol_connector.action_bol_instances').id,